_includes/misc/glossary.md

```diff
@@ -0,0 +1,194 @@
+<dl>
+  <dt>Accessible Rich Internet Applications (ARIA)</dt>
+  <dd>Acronym for <em>Accessible Rich Internet Applications</em> Spell out and describe on first mention within a
+    section.<br />
+    See: <a href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA">MDN Web Docs - ARIA</a>.</dd>
+  <dt>add-on</dt>
+  <dd>Do not use. Use “plugin” instead.</dd>
```

Comment:
do not use add-on?

Reply:
Consistency. We call them plugins everywhere, let's not confuse people.

Reply:
I read this more of a suggestion, so something like this:
```suggestion
<dd>Do not use <em>add-on</em>. Use <em>“plugin”<em> instead.</dd>
```

Reply:
Damn. I thought I had replaced all the double quotes with em elements :(